types/chrome/index.d.ts

```diff
@@ -10223,6 +10261,7 @@ declare namespace chrome.tts {
     export function stop(): void;
     /** Gets an array of all available voices. */
     export function getVoices(callback?: (voices: TtsVoice[]) => void): void;
+    export function getVoices(): Promise<TtsVoice[]>;
```

Comment:
Could you move the Promise version above the callback version? i.e. switch lines 10263 and 10264, to ensure correct behavior by TypeScript.